assets/js/components/product-attribute-control/index.js

```diff
@@ -15,24 +15,16 @@ import { withAttributes } from '../../hocs';
 import ErrorMessage from '../api-error-placeholder/error-message.js';
 import './style.scss';
 
-class ProductAttributeControl extends Component {
-	constructor() {
-		super();
-
-		this.renderItem = this.renderItem.bind( this );
-		this.onExpandAttribute = this.onExpandAttribute.bind( this );
-	}
-
-	onExpandAttribute( item ) {
+const ProductAttributeControl = ( { attributes, error, expandedAttribute, onChange, onOperatorChange, isLoading, operator, selected, termsAreLoading, termsList } ) => {
```

Comment:
This is a long line and should be split into multiline arguments here. [#](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing)

Also applies to other refactors to functional components.

Reply:
How does it sound installing `prettier` and setting it as a pre-commit hook in a follow-up and dealing with these issues there? That's [what we were using in WooCommerce Admin](https://github.com/woocommerce/woocommerce-admin/blob/master/bin/pre-commit-hook.js) and seems like the kind of job better suited for a machine than for a human. :smile: